### DISCUSSION TITLE
Bungeecord support

### DISCUSSION THREAD

**Discussion by sqcred** [Minestom/Minestom#649] [Help] [answered]
How can i add bungeecord support to my server?

**Comment by emortaldev** [comment, marked as answer]
BungeeCordProxy.enable()

Velocity is also supported with VelocityProxy.enable("my_velocity_secret")

**Reply by sqcred** [reply]
thanks but how do i add the skin support?

**Reply by emortaldev** [reply]
Bungeecord should handle skins

**Reply by sqcred** [reply]
oki thanks for the fast support!
